docs/remote_inference_blueprints/bedrock_connector_titan_embedding_blueprint.md

```diff
@@ -89,6 +89,72 @@ POST /_plugins/_ml/connectors/_create
 }
 ```
 
+If you are using AWS OpenSearch Service version 2.11, there are no built-in functions for pre_process_function and post_process_function.
```

Comment:
Clarify the build-in pre/post process functions for Bedrock Titan Embedding model added in 2.12

Reply:
Okay, I think that phrase should go in the part about using built-in functions at the top.